docker/package/tezos_baking_wizard.py

```diff
@@ -513,38 +560,79 @@ def bootstrap_node(self):
         print()
         print("The Tezos node bootstrapped successfully.")
 
+    # Check if an account with the 'baker' alias already exists, and ask the user
+    # if it can be overwritten.
+    def check_baker_account(self):
+        address = subprocess.run(
+            shlex.split("sudo -u tezos tezos-client show address baker --show-secret"),
+            capture_output=True,
+        )
+        if address.returncode == 0:
+            value_regex = b"(?:" + ledger_regex + b")|(?:" + secret_key_regex + b")"
+            value = re.search(value_regex, address.stdout).group(0)
+            address = re.search(address_regex, address.stdout).group(0)
+            print()
+            print("An account with the 'baker' alias already exists.")
+            print("Its current value is", value.decode("utf-8"))
+            print("With a corresponding address:", address.decode("utf-8"))
+
+            return yes_or_no("Should it be replaced with the new import? <y/N> ", "no")
```

Comment:
I guess the message should be changed now to:
```suggestion
            return yes_or_no("Do you wish to import a new key and replace it? <y/N> ", "no")
```
as there is no "the new import" now (the question hasn't been asked yet)